salt/modules/cp.py

```diff
@@ -764,15 +777,16 @@ def push_dir(path, glob=None, upload_path=None):
         return push(path, upload_path=upload_path)
     else:
         filelist = []
-        for root, dirs, files in os.walk(path):
+        for root, files in os.walk(path):
```

Comment:
Thank you, fixed now. I'll go read up on some documentation, my Python seems sluggish...